### DISCUSSION TITLE
Not implemented on Android/IOS

### DISCUSSION THREAD

**Discussion by MaYoRr** [capawesome-team/capacitor-firebase#296] [Q&A] [answered]
@capacitor-firebase/analytics Not implemented on Android and IOS

<img width="801" alt="image" src="https://user-images.githubusercontent.com/49597310/211190538-bf99e2fb-a826-424e-a8c7-fae8a3e32062.png">

`
    "@capacitor-firebase/analytics": "^1.3.0",
    "@capacitor/android": "^4.0.0",
    "@capacitor/app": "^4.1.0",
    "@capacitor/browser": "^4.1.0",
    "@capacitor/core": "^4.0.0",
    "@capacitor/device": "^4.1.0",
    "@capacitor/ios": "^4.0.0",
    "@capacitor/network": "^4.0.1",
    "@capacitor/push-notifications": "^4.1.1",
    "@capacitor/splash-screen": "^4.1.2",
    "@capacitor/toast": "^4.1.0",
    "@devindex/vue-mask": "^2.0.3",
    "@formkit/auto-animate": "^1.0.0-beta.3",
    "@fortawesome/fontawesome-free": "^5.15.4",
    "autoprefixer": "^10",
    "axios": "^1.1.2",
    "capacitor-plugin-safe-area": "^1.0.0",
    "core-js": "^3.8.3",
    "dayjs": "^1.11.6",
    "firebase": "^9.15.0",
    "postcss": "^8",
    "pull-refresh-vue3": "^0.3.1",
    "s-vuex-class": "^0.4.1",
    "swiper": "^8.4.2",
    "tailwindcss": "^3.1.8",
    "vue": "^3.2.13",
    "vue-class-component": "^8.0.0-rc.1",
    "vue-property-decorator": "^10.0.0-rc.3",
    "vue-router": "^4.0.3",
    "vuex": "^4.0.0"
 `

This theme don't work - https://capacitorjs.com/docs/android/troubleshooting#plugin-not-implemented

I call initialization as follows:

  public static async Init() {
    const setEnabled = async () => {
      await FirebaseAnalytics.setEnabled({
        enabled: true,
      });
    };

    const isEnabled = async () => {
      const { enabled } = await FirebaseAnalytics.isEnabled();
      console.log("enabled", enabled);
      return enabled;
    };

    await setEnabled();
    await isEnabled();
  }

**Comment by robingenz** [comment, marked as answer]
Please take a look at the documentation:

> Returns whether or not automatic data collection is enabled.
>
> Only available for Web.

The `isEnabled` method is only available on the Web.

The Firebase Android SDK does not provide a method to query the state. But you can enable and disable it.
